recipes/minmea/all/test_package/test_package.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include "minmea.h"
+
+char* nmea_data[] = {
+"$GPGGA,155246.585,5231.171,N,01321.830,E,1,12,1.0,0.0,M,0.0,M,,*6F",
```

Comment:
Since there has been a shift towards simpler test_package executables lately, this one could probably be reduced to parsing just a single NMEA string as a test as well.